rpm/SPECS/docker-ce-cli.spec

```diff
@@ -50,6 +50,12 @@ pushd /go/src/github.com/docker/cli
 DISABLE_WARN_OUTSIDE_CONTAINER=1 make VERSION=%{_origversion} GITCOMMIT=%{_gitcommit_cli} dynbinary manpages # cli
 popd
 
+if [ "$SUITE" == "7" ]; then
+    yum install -y http://opensource.wandisco.com/centos/7/git/x86_64/wandisco-git-release-7-2.noarch.rpm
+    yum install -y git
+fi
+git version
```

Comment:
more specifically, go mod download is failing because git version on centos7 is 1.8 (anything before git 2.16 is EOL from https://en.wikipedia.org/wiki/Git#Releases). Updating git allows go mod download to succeed, no further issues.
See https://github.com/golang/go/issues/38373 for the go issue and solution. 
I'll update the PR to mention this comment for better documentation in the code

Reply:
> anything before git 2.16 is EOL from https://en.wikipedia.org/wiki/Git#Releases

Yes, upstream releases are EOL, but the CentOS/RHEL versions of Git 1.8 are still maintained (it's kinda horrible, but that's what makes RHEL/CentOS popular as an LTS distro).

I want to avoid installing a non-standard git package in our build-pipeline; especially downloaded over a non-TLS connection. Also not sure if they changed, but I recall (long time ago; when SVN was common) some of the wandisco installers included tracking, which I also want to avoid. (git itself mentions https://repo.ius.io, which could be an alternative).

However, I did a bit more digging for alternatives; the Dockerfile currently uses `GOPROXY=direct`, which (IIRC) was added to allow building from private forks for security releases; https://github.com/docker/docker-ce-packaging/blob/170199f34bc9ca689fd6a97c5c44676a20b8b480/rpm/centos-7/Dockerfile#L9

I did some testing and if we switch to the default (`GOPROXY=https://proxy.golang.org`), Go won't use git to fetch the modules, making the build pass; I pushed a "draft" PR with that change; https://github.com/docker/docker-ce-packaging/pull/530, but can push to your branch if you prefer

Reply:
Thx, I took your update and included GOPROXY here instead of updating git (Indeed I understand we don't want to change the installed packages for the entire CI pipeline). Squashed and added on top of updated scan-cli-plugin version, that simplifies `make` invocation